README.md

```diff
@@ -71,6 +71,20 @@ async fn main() -> Result<(), Error> {
         .try_collect()
         .await?;
 
+    // Or handle the pagination manually.
+    let parameters = SearchParameters::empty().and_raw("_count", "10");  // Set page size.
+    let page: Page<Patient> = client.search_paged(PagedSearchMethod::Parameters(parameters)).await?;
+
+    let patients: Vec<Patient> = page.results;
+    match page.next_url {
+        Some(url) => {
+            let new_page: Page<Patient> = client.search_paged(PagedSearchMethod::Url(url)).await?;
```

Comment:
Yes, it triggers me as well, but the specs say it can be anything, right? If so, I believe we can't do much checks here beyond the `base_url`, maybe we can consider checking against the full `base_url` (as it can contain paths), but its benefit would be limited still.